mbed_lib.json

```diff
@@ -1,6 +1,10 @@
 {
     "name": "ism43362",
     "config": {
+	    "wifi-country_code": {
+            "help": "Country Code to specify cannel number",
+            "value": "\"JP\""
```

Comment:
Why is JP the default value ?

Reply:
I sent you the explanation by mail.

Reply:
Better to explain to everyone. So as I understand, by selecting JP we would scan the 14 channels by default

Reply:
By default, WIFI module is scanning (AT command F3 -> 1 (Set Scan Channel)) the number of channels which is specified by country code. So, by fixing JP (maximum number of channels), we are sure to use efficiently the bandwidth.